nixos/modules/services/hardware/nvidia-container-toolkit/default.nix

```diff
@@ -52,6 +52,15 @@
         '';
       };
 
+      device-name-strategy = lib.mkOption {
+        default = [ "index" ];
```

Comment:
```suggestion
        default = "index";
```

Shouldn't the default be provided like so?

Reply:
yes, thanks for catching that one :)